PCP/src/main/java/PCP/packets/PCPDisconnectionPacket.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * this is a school project under "The Unlicence".
+ */
+package PCP.packets;
+
+import PCP.*;
+import java.util.*;
+
+
+/**
+ *
+ * @author Alessio789
+ */
+public class PCPDisconnectionPacket implements IPCPpacket
+{
+    private byte[] id;
+    private int reason;
```

Comment:
private variable could be a byte. conversion is going to be effetuated in the constructor and in the accessors.

Reply:
I think that this piece of code i right:

- we can handle the id directly in byte[] form
- the reason will be converted in the constructor

Reply:
I was thinking about `reason` but i guess you're right. 
We could also create an enum inside of DisconnectionPacket with the specified opcodes

Reply:
Yes it would be even more orderly.